app/controllers/accounts_controller.rb

```diff
@@ -7,14 +7,17 @@ class AccountsController < ApplicationController
   def show
     respond_to do |format|
       format.html do
+        @pinned_statuses = []
+
         if current_account && @account.blocking?(current_account)
           @statuses = []
           return
         end
 
-        @statuses = filtered_statuses.paginate_by_max_id(20, params[:max_id], params[:since_id])
-        @statuses = cache_collection(@statuses, Status)
-        @next_url = next_url unless @statuses.empty?
+        @pinned_statuses = cache_collection(@account.pinned_statuses.limit(1), Status) unless media_requested?
```

Comment:
In what sort order will this be returned?

Only the first pinned post was displayed.